src/hgraph/_impl/_operators/_conversion_operators/_tsb_conversion_operators.py

```diff
@@ -42,6 +43,15 @@ def combine_named_tsb_partial(tp_: Type[TSB[TS_SCHEMA]] = DEFAULT[OUT], **bundle
     return TSB.from_ts(__type__=tp_, **bundle.as_dict())
 
 
+@graph(overloads=convert)
```

Comment:
Should implement a requirement check to ensure that the schema's are convertible.